src/LanguageServerProtocol.fs

```diff
@@ -113,15 +113,12 @@ module Server =
             JsonRpcError.ErrorDetail(Code = JsonRpcErrorCode.ParseError, Message = ex.Message, Data = data)
           | _ -> ``base``.CreateErrorDetails(request, ex) }
 
-  let startWithSetup<'client when 'client :> Ionide.LanguageServerProtocol.ILspClient>
+  let private startWithSetupCore<'client when 'client :> Ionide.LanguageServerProtocol.ILspClient>
```

Comment:
Any reason to not keep this public? Might be worth having an option for people to pass in whatever handler they'd like.

Reply:
You're right. Made it so.